src/content/extensions/wpgraphql-woocommerce.mdx

```diff
@@ -18,7 +18,8 @@ Using in production is not recommended.
 # Getting Started
 1. Install & activate [WooCommerce](https://woocommerce.com/)
 2. Install & activate WPGraphQL.
-3. Clone or download the [extension](https://github.com/kidunot89/wp-graphql-woocommerce) & activate the **WP GraphQL WooCommerce** plugin
+3. Install & activate [WPGraphQL-JWT-Authentication](https://github.com/wp-graphql/wp-graphql-jwt-authentication).
```

Comment:
You might want to mention this is optional. There are other ways to authenticate and send authorized requests. 

For example, the REST API Basic Auth and REST API Oauth plugins work with WPGraphQL as well, and I believe the REST API JWT one does as well. 

WPGraphQL JWT just allows you to do login mutations to get the tokens